packages/core/saved-objects/core-saved-objects-api-server/src/apis/delete.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { MutatingOperationRefreshSetting, SavedObjectsBaseOptions } from './base';
+
+/**
+ *
+ * @public
+ */
+export interface SavedObjectsDeleteOptions extends SavedObjectsBaseOptions {
+  /** The Elasticsearch Refresh setting for this operation */
+  refresh?: MutatingOperationRefreshSetting;
+  /** Force deletion of an object that exists in multiple namespaces */
```

Comment:
Seeing this again reminds me of the many ways to "break" kibana, especially when a saved object being deleted is referenced by others. 😨